bin/nvcc_wrapper

```diff
@@ -171,7 +171,7 @@ do
     shift
     ;;
   #Handle known nvcc args
-  --dryrun|--verbose|--keep|--keep-dir*|-G|--relocatable-device-code*|-lineinfo|-expt-extended-lambda|--resource-usage|-Xptxas*)
+  --dryrun|--verbose|--keep|--keep-dir*|-G|--relocatable-device-code*|-lineinfo|--expt-extended-lambda|--resource-usage|-Xptxas*)
```

Comment:
If I remember right, this is not a typo.  I think `nvcc` takes this argument with either one or two hyphens.

Reply:
I think the typo is that *we* don't accept either one or two hyphens, right? so it should be something like:

```
--dryrun|--verbose|--keep|--keep-dir*|-G|--relocatable-device-code*|-lineinfo|*-expt-extended-lambda|--resource-usage|-Xptxas*)
```